plugins/search-react/src/components/SearchBar/SearchBar.tsx

```diff
@@ -138,15 +148,19 @@ export const SearchBarBase: ForwardRefExoticComponent<SearchBarBaseProps> =
       );
 
       const endAdornment = (
-        <InputAdornment position="end">
+        <InputAdornment
+          aria-label="Clear search box"
+          sx={{ display: showClearButton }}
+          position="end"
+        >
```

Comment:
An alternative is to assign null to the `endAdorment` when the value is falsy:
```tsx
const endAdornment = value ? (
<InputAdornment position="end">
  <IconButton aria-label="Clear" size="small" onClick={handleClear}>
    <ClearButton />
  </IconButton>
</InputAdornment>
) :  null;
```

Reply:
I agree with Camila here. This is a better solution than the draft I've written before.